stdpopsim/genomes.py

```diff
@@ -2,6 +2,12 @@
 Infrastructure for defining information about species' genomes.
 """
 import attr
+import stdpopsim
+import tskit
+import numpy as np
+import logging
```

Comment:
Imports should usually be grouped following [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) guidelines. (standard library imports first, then third-party imports, and finally local imports). Unfortunately neither flake8 nor black warn about this.

Reply:
is this the right way to import stdpopsim? from within the module?

Reply:
Yep, that should be fine. (And It's the only local import here).